### PR TITLE
[WIP] SideKit Activation method Integration

### PR DESCRIPTION
**Pull request by Spidy123222**
We want to have the ability for app developers to automatically activate jit so that it’s a seamless process to use jit/debug based apps. Adding some jitstreamer attach would give backwards compatibility with some apps that don’t use sidekit also. That means you can also immediately try if it works on apps like provenance and dolphiniOS while developing it also. Url scheme `sidestore://sidejit-enable?bid=BUNDLEIDHERE` 
hopefully at some point we get pid to also work properly instead being a copy of bundled url scheme. 
`sidestore://sidejit-enable?pid=BUNDLEIDHERE` 
is what we have so far and need one for processid and said above jitstreamer attach support. Currently activating jit via url scheme does work. But first activation when the app is asleep it doesn’t activate the debugger for the app. But after second try with app open it works. Might be async causing problems or if it’s because minimuxer wasn’t loaded when it tried.

### Current Changes

- Adds enable jit url scheme for sidekit to use.

### Todo before merge

- [X] Added Url scheme to debug bundleid
- [ ] Make Activation more reliable from external apps. (Seems to not work when trying to wake app on first try which we want to immediately activate when first wake).
- [ ] Add jitstreamer attach pid activation support and sidekit should decide to use this first as it would be fastest and most convenient method with cost it needs sidestore in background still during this and if it doesn’t get scene by the app it will use url scheme. (Jitstreamer attach uses http://69.69.0.1/attach/PIDHERE).
- [ ] Add option in sidestore url scheme to use process Id instead of bundleid.
- [ ] Maybe add a screen to tell its attempting to activate
- [ ] Have error messages popup if there is any
- [ ] Config wireguard to make a 69.69.0.1 local address

### CODE REVIEW

**Comment by naturecodevoid** [comment]
- [ ] minimuxer: add minimuxer_ready function that returns true if last heartbeat was a success and dev image is mounted (@naturecodevoid)
  - [ ] SideStore implementation: possibly an alert "Waiting for SideStore to initialize..." that would be active while we waiting for minimuxer_ready to return true. There would be a cancel button to cancel the operation
- [ ] It would be good to do some sanity checks before executing URL schemes, such as if pairing file is selected

File: AltStore/SceneDelegate.swift [hunk 1]
```diff
@@ -139,7 +141,26 @@ private extension SceneDelegate
...
+                    let v = minimuxer_to_operation(code: 1)
+                    
+                    do {
+                        var x = try debug_app(app_id: jitdebugURLString)
+                        switch x {
+                        case .Good: print(jitdebugURLString)
+                        case .Bad(let code): minimuxer_to_operation(code: code)
+                        }
+                    } catch Uhoh.Bad(let code) {
+                        minimuxer_to_operation(code: code)
```
Comment (naturecodevoid) [inline AltStore/SceneDelegate.swift:159]: Might be good to alert the user if this happens

Comment (Spidy123222) [reply AltStore/SceneDelegate.swift:159]: Ngl that came with the code as I got this from a jitenable swift file. Maybe that should be changed also then. Super new at swift so not sure how to handle this.

Comment (naturecodevoid) [reply AltStore/SceneDelegate.swift:159]: I think you could use a UIAlertController

**Comment by naturecodevoid** [comment]
@Spidy123222 can we delete the branch?

**Pull request closed without merging**